reader.go

```diff
@@ -472,6 +483,11 @@ func decodeLineOfMediaPlaylist(p *MediaPlaylist, wv *WV, state *decodingState, l
 				p.MediaType = VOD
 			}
 		}
+	case strings.HasPrefix(line, "#EXT-X-DISCONTINUITY-SEQUENCE:"):
+		state.listType = MEDIA
+		if _, err = fmt.Sscanf(line, "#EXT-X-DISCONTINUITY-SEQUENCE:%d", &p.DiscontinuitySeq); strict && err != nil {
```

Comment:
Shouldn't you use line slice as below?
line[30:] or 31?
To strip off tag straight ahead.